onnxmltools/convert/sparkml/__init__.py

```diff
@@ -0,0 +1,8 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+from .convert import convert
+from .utils import *
```

Comment:
can we avoid * import here? pylint always suggest we should not use import *

Reply:
I'll change it and make it specific